build-version.sh

```diff
@@ -30,6 +30,22 @@ if [ "$TARGET_ARCH" == "x86_64-unknown-linux-musl" ]; then
 
     CARGO_BIN_DIR="${TEMPDIR}/cargo/bin"
     CRATES2_JSON_PATH="${TEMPDIR}/cargo/.crates2.json"
+elif [ "$TARGET_ARCH" == "aarch64-unknown-linux-gnu" ]; then
+    wget "$(curl -q https://ziglang.org/download/index.json | jq 'to_entries | map([.key, .value])[1][1]["x86_64-linux"] | .tarball' | sed -e 's/^"//' -e 's/"$//')" -O zig -q
+    mkdir -p zigfolder
+    tar -xf ./zig -C zigfolder --strip-components 1
+    rm zig
+    sudo mv zigfolder/zig /usr/local/bin/zig
+    sudo mv zigfolder/lib /usr/local/bin/lib
```

Comment:
* probably just pipe the tarball straight into tar
* can just put the zig dir first in your $PATH rather than copying stuff into /usr/local/bin?


(I'm not sure what `/usr/local/bin/lib` is supposed to be, but it feels wrong)
```suggestion
    mkdir -p zigfolder
    curl "$(curl -q https://ziglang.org/download/index.json | jq 'to_entries | map([.key, .value])[1][1]["x86_64-linux"] | .tarball' | sed -e 's/^"//' -e 's/"$//')"  | tar -x -C zigfolder --strip-components 1

    export PATH="$PWD/zigfolder:$PATH"
```

Reply:
we need /lib or else it'll error with zig not being able to find some lib folder, so yeah the way it was is how we have to do it

Reply:
The error you got from CI was about my broken tar command (I left out the -J)


```suggestion
    mkdir -p zigfolder
    curl "$(curl -q https://ziglang.org/download/index.json | jq 'to_entries | map([.key, .value])[1][1]["x86_64-linux"] | .tarball' | sed -e 's/^"//' -e 's/"$//')"  | tar -xJ -C zigfolder --strip-components 1

    export PATH="$PWD/zigfolder:$PATH"
```

The only reason you need to move the lib folder is that the zig binary is looking relative to itself for its lib foder. If you don't move the zig binary (and instead add it to $PATH) then I would expect this to work just fine.